src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="head_a"
+        href="index.html"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="is-active nav_link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
```

Comment:
The `href` attribute currently has a value of `#`, which is not valid. Consider changing it to a valid anchor link, such as `#photo`.